tests/commands/deploy.spec.ts

```diff
@@ -18,6 +18,7 @@
 import { expect } from 'chai';
 import { promises as fs } from 'fs';
 import * as _ from 'lodash';
+import * as nock from 'nock';
```

Comment:
@toochevere, just a note as I see that this branch/PR is "out of date with the base branch" and test code is being added, while quite a few changes were made to the master branch affecting test code, including bumping the balena-sdk which resulted in new API endpoints being used and required some changes to nock mocked endpoints.
Maybe it won't make any difference to this PR, 🤞  but I suggest rebasing on master sooner than later.

Reply:
thanks for the heads up... will do.